lib/modules/platform/bitbucket-server/utils.ts

```diff
@@ -208,3 +210,13 @@ export function getRepoGitUrl(
   // SSH urls can be used directly
   return cloneUrl.href;
 }
+
+export function getExtraCloneOpts(opts: HostRule): GitOptions {
+  if (opts.token) {
+    addSecretForSanitizing(opts.token, 'global');
```

Comment:
why, i think the token is already sanitized before 🤔

Reply:
True. Just did the same as in Azure Platform without thinking about it